cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/StreamHttpService.java

```diff
@@ -63,6 +63,7 @@ public StreamHttpService(CConfiguration cConf, DiscoveryService discoveryService
       .setHandlerHooks(ImmutableList.of(new MetricsReporterHook(metricsCollectionService,
                                                                 Constants.Stream.STREAM_HANDLER)))
       .setHost(cConf.get(Constants.Stream.ADDRESS))
+      .setPort(cConf.getInt(Constants.Stream.PORT))
```

Comment:
I left the configuration for all ports per your second comment (regarding standalone). Let me know if you'd prefer other services not configurable.